cmd/apmbench/main.go

```diff
@@ -45,13 +45,15 @@ func main() {
 		extraMetrics,
 		resetStoreFunc,
 		Benchmark1000Transactions,
-		BenchmarkOTLPTraces,
 		BenchmarkAgentAll,
 		BenchmarkAgentGo,
 		BenchmarkAgentNodeJS,
 		BenchmarkAgentPython,
 		BenchmarkAgentRuby,
 		Benchmark10000AggregationGroups,
+		BenchmarkOTLPTraces,
+		BenchmarkOTLPLogs,
+		BenchmarkOTLPMetrics,
```

Comment:
Yes it has changed. `_Old` is the old implementation, and the one without suffix is the telemetrygen one. I have yet to understand the meaning of the absolute value. There are subtle differences to the 2 bench implementations (e.g. child spans, batching)

ran against a local apm-server and stack:
```
BenchmarkOTLPTraces      8723683              7473 ns/op
BenchmarkOTLPTraces      9453316              7476 ns/op
BenchmarkOTLPTraces_Old  2873725             25988 ns/op
BenchmarkOTLPTraces_Old  2646979             26076 ns/op
```

Reply:
Hmm, that's a considerable diff. Let's understand the reason for such a huge diff before replacing the older test.

Reply:
I looked into it. Turns out it is due to `sdktrace.WithBlocking()` in old apmbench and `sdktrace.WithBatchTimeout(time.Second)` in telemetrygen. Then there is some minor differences on throughput due to attributes & child spans. As we are copying from telemetrygen, I'm happy to change it to the old WithBlocking. WDYT?

Reply:
As per the docs,

> WithBlocking returns a BatchSpanProcessorOption that configures a BatchSpanProcessor to wait for enqueue operations to succeed instead of dropping data when the queue is full.

I think `WithBlocking` would be required. Also, we can keep or drop the batch timeout -- I will leave it up to you, maybe we can check how it affects the benchmarking result and decide.

Reply:
done. Using WithBlocking only.